globalsSelectors.ipynb

```diff
@@ -38,11 +14,17 @@
    "execution_count": null,
    "outputs": [],
    "source": [
-    "#number of data per visualisation with x axis\n",
-    "global xaxis\n",
-    "xaxis=widgets.Dropdown(options=[('Alls datas',0),('5 per visualisation',5),('25 per visualisation',25),('50 per visualisation',50),('10 first',-1),('10 last',-2)],\n",
-    "                                description='x axe',\n",
-    "                                disabled=False)"
+    "global style\n",
```

Comment:
Great addition! Maybe the selection is a bit complex now, could the dropdown be limited to a selection of all the available styles? (e.g. up to 10 which are different enough between them)